src/core/public/index.ts

```diff
@@ -86,6 +86,7 @@ export { DEFAULT_APP_CATEGORIES, APP_WRAPPER_CLASS } from '../utils';
 export type { AppCategory } from '../types';
 export type {
   UiSettingsParams,
+  PublicUiSettingsParams,
```

Comment:
Sometime tells me `PublicUiSettingsParams` should be named `UiSettingsParams` and exported from the browser-side uiSettings package, but that's out of scope of this PR.